src/app/tests/suites/certification/Test_TC_ACE_1_6.yaml

```diff
@@ -11,662 +11,340 @@
-          [1685009242.317109][39710:39712] CHIP:DMG: status = 0x00 (SUCCESS),
-          [1685009242.317120][39710:39712] CHIP:DMG: },
-          [1685009242.317131][39710:39712] CHIP:DMG:
-          [1685009242.317140][39710:39712] CHIP:DMG: },
-          [1685009242.317159][39710:39712] CHIP:DMG:
-          [1685009242.317167][39710:39712] CHIP:DMG: AttributeStatusIB =
-          [1685009242.317175][39710:39712] CHIP:DMG: {
-          [1685009242.317185][39710:39712] CHIP:DMG: AttributePathIB =
-          [1685009242.317195][39710:39712] CHIP:DMG: {
-          [1685009242.317206][39710:39712] CHIP:DMG: Endpoint = 0x0,
-          [1685009242.317218][39710:39712] CHIP:DMG: Cluster = 0x1f,
-          [1685009242.317229][39710:39712] CHIP:DMG: Attribute = 0x0000_0000,
-          [1685009242.317243][39710:39712] CHIP:DMG: ListIndex = Null,
-          [1685009242.317251][39710:39712] CHIP:DMG: }
-          [1685009242.317265][39710:39712] CHIP:DMG:
-          [1685009242.317272][39710:39712] CHIP:DMG: StatusIB =
-          [1685009242.317281][39710:39712] CHIP:DMG: {
-          [1685009242.317291][39710:39712] CHIP:DMG: status = 0x00 (SUCCESS),
-          [1685009242.317301][39710:39712] CHIP:DMG: },
-          [1685009242.317310][39710:39712] CHIP:DMG:
-          [1685009242.317320][39710:39712] CHIP:DMG: },
-          [1685009242.317336][39710:39712] CHIP:DMG:
-          [1685009242.317342][39710:39712] CHIP:DMG: AttributeStatusIB =
-          [1685009242.317353][39710:39712] CHIP:DMG: {
-          [1685009242.317362][39710:39712] CHIP:DMG: AttributePathIB =
-          [1685009242.317373][39710:39712] CHIP:DMG: {
-          [1685009242.317384][39710:39712] CHIP:DMG: Endpoint = 0x0,
-          [1685009242.317398][39710:39712] CHIP:DMG: Cluster = 0x1f,
-          [1685009242.317406][39710:39712] CHIP:DMG: Attribute = 0x0000_0000,
-          [1685009242.317417][39710:39712] CHIP:DMG: ListIndex = Null,
-          [1685009242.317427][39710:39712] CHIP:DMG: }
-          [1685009242.317441][39710:39712] CHIP:DMG:
-          [1685009242.317449][39710:39712] CHIP:DMG: StatusIB =
-          [1685009242.317460][39710:39712] CHIP:DMG: {
-          [1685009242.317471][39710:39712] CHIP:DMG: status = 0x00 (SUCCESS),
-          [1685009242.317481][39710:39712] CHIP:DMG: },
-          [1685009242.317491][39710:39712] CHIP:DMG:
-          [1685009242.317502][39710:39712] CHIP:DMG: },
-          [1685009242.317515][39710:39712] CHIP:DMG:
-          [1685009242.317523][39710:39712] CHIP:DMG: ],
-          [1685009242.317544][39710:39712] CHIP:DMG:
-          [1685009242.317551][39710:39712] CHIP:DMG: InteractionModelRevision = 1
-          [1685009242.317558][39710:39712] CHIP:DMG: }
-          [1685009242.317637][39710:39712] CHIP:DMG: WriteClient moving to [AwaitingDe]
-      disabled: true
+      command: "writeAttribute"
+      attribute: "ACL"
+      arguments:
+          value: [
+                  {
+                      FabricIndex: FabricIndexValue,
```

Comment:
Again, just use 0 for all the FabricIndex bits.

Reply:
Updated.